dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -91,6 +93,26 @@ dashboard.buildHeader = function (stageData, progressData, currentLevelId) {
   $('.header_popup').click(function (e) {
     e.stopPropagation(); // Clicks inside the popup shouldn't close it
   });
+  $('.header_popup_close').click(hideHeaderPopup);
+
+  $(window).resize(function () {
+    if ($('.header_popup').is(':visible')) {
```

Comment:
Nit: Maybe use a var instead since `resize` can fire a lot, and we'll end up doing a DOM lookup each time to check visibility.